### ISSUE TITLE
All submitting of text

### ISSUE THREAD

**Issue by atmos** [atmos/peonies#6]
Right now we're just displaying a stream of events, let's make it possible to submit text and start using this thing.

**Comment by technicalpickles** [comment]
This is maybe a little hard :sweat: 

`react/Input` is where the [submit handling happens](https://github.com/atmos/electrogram/blob/master/src/react/input.coffee#L8-L10). It has [access to the `models/App`](https://github.com/atmos/electrogram/blob/master/src/main.coffee#L11). `models/App` has access to the [slack connections](https://github.com/atmos/electrogram/blob/master/src/models/app.coffee#L12) in order to send a message, and it has the [`react/App` instance](https://github.com/atmos/electrogram/blob/master/src/models/app.coffee#L13) so it can know [the active channel](https://github.com/atmos/electrogram/blob/master/src/react/app.coffee#L16).

Right now there is only one input field globally, so I was wondering if making each channel have it's own input would simplify this, in addition to giving an independent input area between channels.

**Comment by atmos** [comment]
Yup, I finally found https://facebook.github.io/react/docs/thinking-in-react.html and think that approach is a lot more sane. We need things to bubble up, not descend down the object hierarchy.

**Comment by technicalpickles** [comment]
I was trying to do some more reading on react, and came across [flux](http://facebook.github.io/flux/) (and here's [writeup](http://blog.andrewray.me/flux-for-stupid-people/)). Not sure if it's specifically required, but might be useful in thinking about the architecture.
